### DISCUSSION TITLE
Lesson 10: Getting real world price data from Chainlink

### DISCUSSION THREAD

**Discussion by sandeepV2** [Cyfrin/foundry-full-course-cu#1702] [Q&A]
Failing to fetch lastRoudData.

Error on execution:

>

>  {
> 	"error": "Failed to decode output: Error: data out-of-bounds (length=0, offset=32, code=BUFFER_OVERRUN, version=abi/5.7.0)"
> }

https://github.com/smartcontractkit/documentation/issues/1940

**Comment by EngrPips** [comment]
Hello @sandeepV2, I answered here: https://github.com/smartcontractkit/documentation/issues/1940 with my result when I tried Getting the real world price data from Chainlink.
